From c7ea870e08684d43529892a3ea1ec1834ba9f007 Mon Sep 17 00:00:00 2001
From: Valentine Barshak <valentine.barshak@cogentembedded.com>
Date: Fri, 9 Aug 2019 00:41:42 +0300
Subject: [PATCH 03/21] configs: rcar-gen3-common: Increase
 CONFIG_SYS_MALLOC_LEN to 8MB

This increases CONFIG_SYS_MALLOC_LEN from 1MB to 8MB,
otherwise ext4write may run out of memory on big ext4
partitions. Looks like 8MB is enough for 64GB partitions.

Signed-off-by: Valentine Barshak <valentine.barshak@cogentembedded.com>
---
 include/configs/rcar-gen3-common.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/rcar-gen3-common.h b/include/configs/rcar-gen3-common.h
index 0ce87b8..b6eac37 100644
--- a/include/configs/rcar-gen3-common.h
+++ b/include/configs/rcar-gen3-common.h
@@ -47,7 +47,7 @@
 
 #define CONFIG_SYS_MONITOR_BASE		0x00000000
 #define CONFIG_SYS_MONITOR_LEN		(1 * 1024 * 1024)
-#define CONFIG_SYS_MALLOC_LEN		(1 * 1024 * 1024)
+#define CONFIG_SYS_MALLOC_LEN		(8 * 1024 * 1024)
 #define CONFIG_SYS_BOOTM_LEN		(64 << 20)
 
 /* The HF/QSPI layout permits up to 1 MiB large bootloader blob */
-- 
2.7.4

